BedrockServer.cpp

```diff
@@ -1548,3 +1609,16 @@ void BedrockServer::_beginShutdown(const string& reason, bool detach) {
 bool BedrockServer::backupOnShutdown() {
     return _backupOnShutdown;
 }
+
+SData BedrockServer::_generateBlacklistMessage(const BedrockCommand& command) {
```

Comment:
I'm not sure why this is its own function; it seems like you would always want to just call this, send immediately, and then -- actually, not exit, just start flushing the buffers until you have been able to confirm you've sent it, and then exit.